llvm/test/CodeGen/MIR/AMDGPU/init-whole.wave.mir

```diff
@@ -0,0 +1,17 @@
+; RUN: llc -global-isel=0 -march=amdgcn -mcpu=gfx1100 -stop-after=amdgpu-isel < %s | FileCheck --check-prefix=GCN %s
+; RUN: llc -global-isel=1 -march=amdgcn -mcpu=gfx1100 -stop-after=amdgpu-isel < %s | FileCheck --check-prefix=GCN %s
```

Comment:
-stop-after=amdgpu-isel won't work correctly for globalisel (but this check will work no matter where it stops). Might as well use -stop-after=finalize-isel

Reply:
Done